articles/beta-changelog.mdx

```diff
@@ -16,87 +16,88 @@ explicit_canonical:
 
-## Workflow updates
+## Conflicting changes
 
-* Improved the process for uploading, moving, renaming, and deleting files is consistent with Git workflows.
-* Selecting *Upload*, *Move*, *Rename*, or *Delete* no longer immediately performs that action. Instead, the file or folder will be marked and that action will occur the next time you save your changes. Marked files will receive a *New*, *Edited*, or *Deleted* badge to indicate they have unsaved changes.
-* You can now move, rename, or delete a file or folder by selecting that option in the *Context menu* in the top right of the file card or the *File menu* in the top left of the editing interface.
-* You can restore deleted files before you save your site by selecting *Restore file* in the *Context menu* in the top right of the file card, in the *File menu* in the top left of the editing interface, or in the banner at the bottom of the editing interface.
+* CloudCannon now detects and blocks incoming changes that conflict with your unsaved changes. This will prevent your save from replacing any updates from an external source (i.e., race condition).
+* You will be notified when your site has *Conflicting changes* via a warning message in your *Site navigation* and *Review changes* modal. You can address these *Conflicting changes* by saving the affected files to attempt a merge or by discarding unsaved changes for those files to remove the conflict.
 
-## *Context menu* and *File menu* improvements
+## Badge improvements
 
-* The *Context menu* in the top right of a file card and the *File menu* in the top left of the editing interface are now consistent. Both menus now include the *Publish to posts* (Jekyll only), *Move*, *Clone*, *Rename*, *Download*, *Discard unsaved changes*, and *Delete* file actions.
-* The recently opened files list has been removed from the *File menu* in the top left of the editing interface.
-* You can now navigate the *Context menu* using the arrow, tab, and escape keys on your keyboard.
+* Three badge types appear on file and folder cards, indicating unsaved changes. These badges are *New*, *Edited*, and *Deleted.* These badges are removed from cards once all changes have been saved.
+* The *Edited* badge indicates a file’s content has been updated or files within a folder have been added, edited, or marked for deletion. The *Edited* badge will also appear on cards on the *Sites list* page when there are unsaved changes for that site.
+* The *New* badge indicates a file or folder was recently created or that all files within a folder were recently created.
+* The *Deleted* badge indicates a file or folder has been marked for deletion or that all files within a folder have been marked for deletion.
```

Comment:
```suggestion
* The *Deleted* badge indicates a file has been marked for deletion or that all files within a folder have been marked for deletion.
```

Deleting all files within a folder is the same case as "deleting the folder". Not hugely important, but you could avoid mentioning it twice here.